pyproject.toml

```diff
@@ -73,6 +73,7 @@ cron-descriptor = {version = ">=1.2.32", optional = true}
 pipdeptree = {version = ">=2.9.0,<2.10", optional = true}
 pyathena = {version = ">=2.9.6", optional = true}
 weaviate-client = {version = ">=3.22", optional = true}
+adlfs = {version = "^2023.8.0", optional = true}
```

Comment:
please set the same version as all other fsspec components, they are pinned strict to the same version